src/Aspire.Hosting/DistributedApplicationExecutionContext.cs

```diff
@@ -36,7 +36,7 @@ public DistributedApplicationExecutionContext(DistributedApplicationOperation op
     /// <summary>
     /// The name of the publisher that is being used if <see cref="Operation"/> is set to <see cref="DistributedApplicationOperation.Publish"/>. 
     /// </summary>
-    public string? PublisherName { get; set; }
+    public string PublisherName { get; set; }
```

Comment:
@mitchdenny This one was not sure but since there is no way to pass a `null` values from the ctors. Unless we expect someone to set the value afterwards.

Reply:
If you don't pass a publisher name then it is null - which is valid if you are not in publisher mode.

Reply:
```c#
public DistributedApplicationExecutionContext(DistributedApplicationOperation operation) : this(operation, "manifest")
```

won't be null, right?